integration/experiment/sst_evaluation/run_sst_evaluation_nas_bt.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
+#
+#  Copyright (c) 2015 - 2023, Intel Corporation
+#  SPDX-License-Identifier: BSD-3-Clause
+#
+
+'''
+Run a power sweep with NPB BT.
+'''
+
+import argparse
+
+from experiment.sst_evaluation import sst_evaluation
+from experiment import machine
+from apps.nasbt import nasbt
```

Comment:
nasbt does not exist in dev, nor was it introduced in this PR.

Reply:
I think https://github.com/geopm/geopm/pull/2926 should cover it. I'll try rebasing that one on the latest dev.

Reply:
I just rebased #2926 on dev and am reviewing it first since this PR has a dependency on it.